app/assets/javascripts/schedules.js

```diff
@@ -1,10 +1,14 @@
 $(document).ready(function() {
   $("#my_calendar").click(function(){
     $(".menu_my_calendar").slideToggle();
+    var ico_schedule = $(".ico-schedule");
```

Comment:
Actually, you don't need define variable ico_schedule. You can send directly to argument `changeIcon` function